zerver/models.py

```diff
@@ -1499,7 +1502,8 @@ def get_user_including_cross_realm(email, realm=None):
 @cache_with_key(bot_profile_cache_key, timeout=3600*24*7)
 def get_system_bot(email):
     # type: (Text) -> UserProfile
-    return UserProfile.objects.select_related().get(email__iexact=email.strip())
+    return UserProfile.objects.select_related().get(email__iexact=email.strip(),
+                                                    realm=get_realm("zulip"))
```

Comment:
This change isn't quite correct, because the system bots may or may not be in the `zulip` realm.